Backend/Models/Database/Medicament.cs

```diff
@@ -51,5 +52,8 @@ public class Medicament
 
         [Required]
         public PharmaceuticalForm PharmaceuticalForm { get; set; }
+
+        [Required]
+        public ICollection<RequiredMedicamentAmount> RequiredMedicamentAmounts { get; set; }
```

Comment:
According to our class diagram, this relation is optional, so we do not need `[Required]` field.

Reply:
Fixed